.github/workflows/build.yml

```diff
@@ -0,0 +1,205 @@
+name: build
+
+on:
+  push:
+    branches: [ master, '[0-9]+.[0-9]' ]
+  pull_request:
+    branches: [ master, '[0-9]+.[0-9]' ]
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip travis]')"
+    name: "PHP ${{ matrix.php-versions }} Test ${{ matrix.static-analysis != 'no' && matrix.static-analysis || '' }} (deps: ${{ matrix.dependencies }})"
+
+    continue-on-error: ${{ matrix.experimental }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.1', '7.2', '7.3', '7.4']
```

Comment:
7.1 was not tested by travis. Do we need it here?

Reply:
Good point. According to https://docs.neos.io/cms/installation-development-setup/system-requirements#php-version-matrix we promise only 72+

Reply:
According to https://github.com/neos/flow-development-collection/pull/2390/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L10 we did run 7.1 builds on 5.3 and in composer it's also marked as ~7.1 (https://github.com/neos/flow-development-collection/blob/5.3/composer.json#L9), but we can surely strip the build.

Reply:
No, then let's leave it in!